index.php

```diff
@@ -168,6 +179,10 @@ function display(){
 
 <input class="input input--full-width" type="text" name="phone" placeholder="Phone Number" required>
 <br>
+<input class="input" type="checkbox" name="omitgrade" value="True" >I only want a notification that my grades are available. The text can omit the actual grade I recieved.</input>
```

Comment:
What do you think about either
1. removing the last line and adding a footer that says: "The text will omit the actual grade you receive "
2. Switching the wording to "The text _should_ omit the actual grade I received"